app/scripts/directives/deploymentDonut.js

```diff
@@ -53,6 +56,23 @@ angular.module('openshiftConsole')
         $scope.$watchGroup(['limitRanges', 'hpa', 'project'], updateHPAWarnings);
         $scope.$watch('rc.spec.template.spec.containers', updateHPAWarnings, true);
 
+        var updateQuotaWarning = function() {
+          if (_.get($scope.rc, 'spec.replicas', 0) > _.get($scope.rc, 'status.replicas', 0)) {
```

Comment:
RC `spec.replicas` defaults to 1 if not set.

https://github.com/spadgett/origin/blob/fc85c362316ba6c7cd01341da2a2f27c186bdc6b/vendor/k8s.io/client-go/1.4/pkg/api/v1/types.go#L2011-L2015